### ISSUE TITLE
Using the 'Detail view' to show the all message payloads in Saga View

### ISSUE THREAD

**Issue by HEskandari** [Particular/ServiceInsight#641]
- Showing the 'Detail view' for the all objects (in the middle panel and right / left hand panels) as a generic way to display saga data. 
- The threshold and use of ellipsis when the content gets long should be removed.
- User will be able to see all the trimmed property names and values in the detail view.

This is a split of #624.

**Comment by adamralph** [comment]
This is labelled as a bug, but I'm finding it hard to parse the description to know what the symptom of the bug is, and how it should be fixed. Can we improve the description somehow?

**Comment by HEskandari** [comment]
@adamralph I no longer think this should be labelled as a bug. It is an improvement on how the message payloads are displayed. Currently they are displayed in expandable panels, all loaded at once when clicking the Show Message button.  I'll update the description and add what I think should be the expected behavior.

**Comment by adamralph** [comment]
Thanks @HEskandari. I made another tweak to the title so it reads more like a feature (e.g. in release notes).

**Comment by HEskandari** [comment]
Reviewed. This is still the case, but requires substantial work to redesign the view.
